requirements/base.in

```diff
@@ -18,3 +18,7 @@
 -e file:.
 pyrsistent>=0.16.1,<0.17
 zipp==3.4.1
+click
+py>=1.10.0
+urllib3>=1.26.5
```

Comment:
I'm not clear why this included here and not in `setup.py`. Also [this](https://github.com/apache/superset/blob/9773aba522e957ed9423045ca153219638a85d2f/superset/db_engine_specs/clickhouse.py#L51) non-top level import of `urllib3` seems weird.